sdk/ai/Azure.AI.Projects.OneDP/samples/Sample5_Evaluations.md

```diff
@@ -0,0 +1,100 @@
+# Sample for Azure.AI.Projects.OneDP with `Evaluations`
+
+In this example, we will demonstrate creating, listing and retrieving evaluations using the `Evaluations` client in `Azure.AI.Projects.OneDP`. This uses a Dataset as the input data for the evaluation and an Evaluator ID for the evaluation type.
```

Comment:
`Azure.AI.Projects` instead of `Azure.AI.Projects.OneDP` (the name will change shortly)